superflore/generators/bitbake/run.py

```diff
@@ -73,7 +72,7 @@ def main():
         selected_targets = [args.ros_distro]
         preserve_existing = False
     if not selected_targets:
-        selected_targets = active_distros + ros2_distros
+        selected_targets = get_distros_by_status('active')
```

Comment:
Just a note, this will not get `dashing` at the moment as it's currently in `prerelease`.

Reply:
I believe that's okay as its [distribution](https://raw.githubusercontent.com/ros/rosdistro/master/dashing/distribution.yaml) is not complete anyway. We still can specify it by using ```--ros-distro```.